be/src/storage/tablet_manager.cpp

```diff
@@ -1576,8 +1576,9 @@ void TabletManager::get_tablets_basic_infos(int64_t table_id, int64_t partition_
                 auto& tablet = itr.second;
                 auto table_id_in_meta = tablet->tablet_meta()->table_id();
                 if ((table_id == -1 || table_id_in_meta == table_id) &&
-                    (authorized_table_ids != nullptr &&
-                     authorized_table_ids->find(table_id_in_meta) != authorized_table_ids->end())) {
+                    (authorized_table_ids == nullptr ||
+                     (authorized_table_ids != nullptr &&
+                      authorized_table_ids->find(table_id_in_meta) != authorized_table_ids->end()))) {
```

Comment:
```suggestion
                    (authorized_table_ids == nullptr ||
                     authorized_table_ids->find(table_id_in_meta) != authorized_table_ids->end())) {
```
in this case, `authorized_table_ids != nullptr` is implicitly included.

Reply:
You're right. i'm done